stable/postgresql/values-production.yaml

```diff
@@ -257,6 +257,9 @@ resources:
   requests:
     memory: 256Mi
     cpu: 250m
+  limits:
```

Comment:
Thanks for the PR! I think it is a good feature to add to the chart. However, I don't think we should specify any value by default for the limit. I would just leave the limit option commented.

Apart from that, note the [circleci check](https://circleci.com/gh/helm/charts/69007?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link) is failing because you have to increase the chart version in the Chart.yaml file.

Reply:
Thanks for the review @tompizmor! I will make these changes to the PR ASAP.

Reply:
Ok!